dev-requirements.txt

```diff
@@ -14,8 +14,7 @@ Sphinx==1.6.5
 sphinx_rtd_theme
 flasgger==0.9.1
 -e git+https://git@github.com/uc-cdis/cdisutils-test.git@0.0.1#egg=cdisutilstest
```

Comment:
@paulineribeyre do you know if `cdisutils-test` need to be updated?

Reply:
cdisutilstest has already been updated python 3 (version 1.0.0 🎉)

Reply:
I'll bump it to 1.0.0 here then.